{"sections":[],"hierarchy":{"paths":[["doc:\/\/BlueConnect\/documentation\/BlueConnect","doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError"]]},"metadata":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}],"title":"BleCentralManagerProxyError.connectionTimeout","externalID":"s:11BlueConnect27BleCentralManagerProxyErrorO17connectionTimeoutyA2CmF","role":"symbol","symbolKind":"case","roleHeading":"Case","modules":[{"name":"BlueConnect"}]},"schemaVersion":{"minor":3,"major":0,"patch":0},"kind":"symbol","primaryContentSections":[{"declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}]}],"kind":"declarations"},{"content":[{"type":"heading","anchor":"discussion","level":2,"text":"Discussion"},{"inlineContent":[{"type":"text","text":"This error is thrown when a peripheral connection attempt exceeds the allowed time limit."}],"type":"paragraph"}],"kind":"content"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout"]}],"abstract":[{"text":"Indicates that a timeout occurred during peripheral connection.","type":"text"}],"identifier":{"url":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","interfaceLanguage":"swift"},"references":{"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError/connectionTimeout":{"role":"symbol","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror\/connectiontimeout","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError\/connectionTimeout","abstract":[{"type":"text","text":"Indicates that a timeout occurred during peripheral connection."}],"kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectionTimeout","kind":"identifier"}],"title":"BleCentralManagerProxyError.connectionTimeout","type":"topic"},"doc://BlueConnect/documentation/BlueConnect/BleCentralManagerProxyError":{"type":"topic","abstract":[{"text":"An enumeration representing various errors that can occur in the ","type":"text"},{"code":"BleCentralManagerProxy","type":"codeVoice"},{"text":".","type":"text"}],"navigatorTitle":[{"kind":"identifier","text":"BleCentralManagerProxyError"}],"role":"symbol","url":"\/documentation\/blueconnect\/blecentralmanagerproxyerror","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect\/BleCentralManagerProxyError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"BleCentralManagerProxyError"}],"kind":"symbol","title":"BleCentralManagerProxyError"},"doc://BlueConnect/documentation/BlueConnect":{"title":"BlueConnect","abstract":[],"kind":"symbol","role":"collection","type":"topic","url":"\/documentation\/blueconnect","identifier":"doc:\/\/BlueConnect\/documentation\/BlueConnect"}}}